investd/config.py

```diff
@@ -11,14 +11,15 @@
 from pathlib import Path
 from typing import Any
 
-from .common import Currency
+from investd.common import Currency
 
 log = logging.getLogger(__name__)
 
-SOURCE_BASE_PATH = Path(os.getenv("SOURCE_BASE_PATH", "data/source"))
-PERSIST_PATH = Path(os.getenv("PERSIST_PATH", "data/persist"))
-REPORTS_PATH = Path(os.getenv("REPORTS_PATH", "data/reports"))
-REF_CURRENCY = Currency(os.getenv("REF_CURRENCY", "USD"))
+INVESTD_DATA = Path(os.getenv("INVESTD_DATA", "investd_data"))
+INVESTD_SOURCES = Path(os.getenv("INVESTD_SOURCES", INVESTD_DATA / "sources"))
```

Comment:
sample_data has "source" instead of "sources"

Reply:
Renamed